Objects/Converters/ConverterRevit/ConverterRevitShared/Partial Classes/ConvertAnalyticalStick.cs

```diff
@@ -48,30 +45,38 @@ public List<ApplicationPlaceholderObject> AnalyticalStickToNative(Element1D spec
           RevitColumn revitColumn = new RevitColumn();
           revitColumn.type = speckleStick.property.name.Replace('X', 'x');
           revitColumn.baseLine = speckleStick.baseLine;
           placeholders = ColumnToNative(revitColumn);
           DB.FamilyInstance nativeRevitColumn = (DB.FamilyInstance)placeholders[0].NativeObject;
-          AnalyticalModelColumn analyticalModelCol = (AnalyticalModelColumn)nativeRevitColumn.GetAnalyticalModel();
-          analyticalModelCol.SetReleases(true, Convert.ToBoolean(speckleStick.end1Releases.stiffnessX), Convert.ToBoolean(speckleStick.end1Releases.stiffnessY), Convert.ToBoolean(speckleStick.end1Releases.stiffnessZ), Convert.ToBoolean(speckleStick.end1Releases.stiffnessXX), Convert.ToBoolean(speckleStick.end1Releases.stiffnessYY), Convert.ToBoolean(speckleStick.end1Releases.stiffnessZZ));
-          analyticalModelCol.SetReleases(false, Convert.ToBoolean(speckleStick.end2Releases.stiffnessX), Convert.ToBoolean(speckleStick.end2Releases.stiffnessY), Convert.ToBoolean(speckleStick.end2Releases.stiffnessZ), Convert.ToBoolean(speckleStick.end2Releases.stiffnessXX), Convert.ToBoolean(speckleStick.end2Releases.stiffnessYY), Convert.ToBoolean(speckleStick.end2Releases.stiffnessZZ));
-          analyticalModelCol.SetOffset(AnalyticalElementSelector.StartOrBase, offset1);
-          analyticalModelCol.SetOffset(AnalyticalElementSelector.EndOrTop, offset2);
+          SetAnalyticalPros(nativeRevitColumn, speckleStick, offset1, offset2);
           return placeholders;
           //Column column = new Column(speckleStick.baseLine);
           return ColumnToNative(revitColumn);
       }
       return placeholderObjects;
     }
 
+    private void SetAnalyticalPros(Element element, Element1D element1d, XYZ offset1, XYZ offset2)
+    {
+#if !REVIT2023
+      var analyticalModel = (AnalyticalModelStick)element.GetAnalyticalModel();
+      analyticalModel.SetReleases(true, Convert.ToBoolean(element1d.end1Releases.stiffnessX), Convert.ToBoolean(element1d.end1Releases.stiffnessY), Convert.ToBoolean(element1d.end1Releases.stiffnessZ), Convert.ToBoolean(element1d.end1Releases.stiffnessXX), Convert.ToBoolean(element1d.end1Releases.stiffnessYY), Convert.ToBoolean(element1d.end1Releases.stiffnessZZ));
+      analyticalModel.SetReleases(false, Convert.ToBoolean(element1d.end2Releases.stiffnessX), Convert.ToBoolean(element1d.end2Releases.stiffnessY), Convert.ToBoolean(element1d.end2Releases.stiffnessZ), Convert.ToBoolean(element1d.end2Releases.stiffnessXX), Convert.ToBoolean(element1d.end2Releases.stiffnessYY), Convert.ToBoolean(element1d.end2Releases.stiffnessZZ));
+      analyticalModel.SetOffset(AnalyticalElementSelector.StartOrBase, offset1);
+      analyticalModel.SetOffset(AnalyticalElementSelector.EndOrTop, offset2);
+#else
+      var analyticalModel = Doc.GetElement(AnalyticalToPhysicalAssociationManager.GetAnalyticalToPhysicalAssociationManager(Doc).GetAssociatedElementId(element.Id)) as AnalyticalMember;
```

Comment:
Could not find a way to get and set the offsets in the new API!

Reply:
Seems like they just moved the method out of the analytical model. This offset is only used for context of BIM in particular from a Revit POV. We can use the offset from the base element instead if it's missing